src/FsToolkit.ErrorHandling.TaskResult/Task.fs

```diff
@@ -45,6 +45,9 @@ module Task =
 
     let inline map3 ([<InlineIfLambda>] f) x y z = apply (map2 f x y) z
 
+    let inline ignore (x: Task<'a>) =
```

Comment:
I'd go with copying
https://github.com/fsprojects/FSharp.Control.TaskSeq/blob/main/src/FSharp.Control.TaskSeq/Utils.fs#L45-L51

Abel would know the details as to exactly why...